src/stache/includes/cli/no-lint.html

```diff
@@ -0,0 +1 @@
+<sky-code>--no-lint</sky-code> — Prevents the <a href="https://palantir.github.io/tslint/">TSLint static analysis tool</a> from validating TypeScript code in the <sky-code>src</sky-code> folder against the rules specified by <a href="https://github.com/blackbaud/skyux-builder/blob/master/tslint.json">the <sky-code>tslint.json</sky-code> file in the SKY UX Builder repo</a>.
```

Comment:
Technically this is only true if the `tslint.json` file in the project's root references builder in it's `extends` property, as seen in the template.  https://github.com/blackbaud/skyux-sdk-template/blob/master/tslint.json#L2

I think we should just say based on your projects `tslint.json` file.

Reply:
I updated this and also similar text at https://developer.blackbaud.com/skyux/learn/reference/cli-commands/lint.